src/components/soknadsStatus/SoknadsStatus.tsx

```diff
@@ -116,40 +115,35 @@ const HeadingWrapper = styled.div`
     text-align: center;
 `;
 
+const leserData = (restStatus: REST_STATUS): boolean => {
+    return (
+        restStatus === REST_STATUS.INITIALISERT || restStatus === REST_STATUS.PENDING || restStatus === REST_STATUS.OK
+    );
+};
+
```

Comment:
skal leserData returnere true hvis reststatus er OK?

Reply:
Hmm, så den blir bare brukt for å sette hasError? Kanskje kalle funskjonen noe annet da? Fordi dette har ikke noe med å lese data å gjøre?

Reply:
Har også sett det flere plasser hvor leserData er rett å slett bare en restStatus sjekk. Men har ikke tenkt så mye på det. Men ja, jeg tar å endre det til noe annet som er mer forklarende.

Reply:
supert. Og sikkert lurt å ta en dobbeltsjekk på at den bare blir brukt til å vise feil andre steder også!

Reply:
Skal gjøre!